epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -53,69 +55,82 @@
+				publications can be created with a high degree of accessibility simply through the proper application of
+				established digital accessibility techniques.</p>
 
-			<p>In order to outline how to make fully accessible EPUBs, the <a href="https://www.w3.org/TR/epub-a11y-11/"
-					>EPUB Accessibility</a> guidelines have been drafted to address two key needs in the EPUB
-				ecosystem:</p>
+			<p>The <a href="https://www.w3.org/TR/epub-a11y-11/">EPUB Accessibility</a> specifications [epub-a11y-11] address the following key areas for making fully accessible EPUB publications:</p>
 
 			<ul>
 				<li>discoverability of accessibility features of publications;</li>
-				<li>evaluation and certification of accessible EPUB publications.</li>
+				<li>conformance to Web Content Accessibility Guidelines;</li>
+				<li>specific EPUB requirements for accessibility;</li>
+				<li>evaluation and conformance reporting of accessible EPUB publications.</li>
 			</ul>
-
-			<p>Providing accessibility metadata facilitates informed decisions about the usability of an EPUB
-				publication. Consumers can examine the characteristics of the content and decide whether an EPUB
-				publication is appropriate for each need.</p>
-
-			<p>While it has always been possible to create EPUB publications with a high degree of accessibility, this
-				document also establishes formal requirements for contents to be certified as accessible. These
-				requirements provide EPUB creators with a clear set of guidelines for evaluating their content with
-				respect to quality certification and enable it.</p>
-
+			
+			<p>The EPUB Accessibility defines requirements for accessibility metadata. In this way consumers can examine the accessibility characteristics of the content and make well-informed decisions on whether an EPUB publication is appropriate for each need before buying, borrowing or downloading it.
+				This metadata, attached to the digital file and distributed along the supply chain, ensures that persons with disabilities can be informed about the publication accessibility features.
+				In order to make sure that information gets to the end user, it is critical that all players in the value chain do their part in managing metadata and displaying it to the end user (aggregators, ecommerce, reading solutions).</p>
+
+			<p>EPUB Accessibility also defines the official accessibility criteria that EPUBs must adhere to claim conformance to the standard.
+				These criteria offer EPUB creators a precise set of instructions to assess their content in terms of quality assurance.</p>
+			
 			<p>The EPUB Accessibility does not target a single version of EPUB (most commercially available publications
 				are in EPUB 3 or EPUB 2 format). It is designed to be applicable to all EPUB publications conforming to
 				any version or profile, including future versions of the standard.</p>
+				
+			<p>The EPUB allows to create both reflowable and Fixed Layout publications.</p>
```

Comment:
"The EPUB" is confusing. This should probably be "EPUB 3", or a specific reference to 3.3 with a citation.